WORKSPACE.yaml

```diff
@@ -17,6 +17,8 @@ defaultArgs:
   phpstormDownloadUrl: "https://download.jetbrains.com/webide/PhpStorm-2022.2.3.tar.gz"
   rubymineDownloadUrl: "https://download.jetbrains.com/ruby/RubyMine-2022.2.3.tar.gz"
   webstormDownloadUrl: "https://download.jetbrains.com/webstorm/WebStorm-2022.2.3.tar.gz"
+  riderDownloadUrl: "https://download.jetbrains.com/rider/JetBrains.Rider-2022.2.3.tar.gz"
+  clionDownloadUrl: "https://download.jetbrains.com/cpp/CLion-2022.2.3.tar.gz"
```

Comment:
We can update here to v2022.2.4:

<img width="424" alt="image" src="https://user-images.githubusercontent.com/418083/202692982-4aa32b2f-80cb-4258-8f9b-aa8cf5e3f8a9.png">

<img width="417" alt="image" src="https://user-images.githubusercontent.com/418083/202692885-49a821aa-235f-40bf-95f3-80d035440b2a.png">

Noticed after seeing this "update" label:
<img width="287" alt="image" src="https://user-images.githubusercontent.com/418083/202692834-1ddf8a43-d714-44fe-b0c2-a6c031df51c5.png">

Reply:
- https://github.com/gitpod-io/gitpod/pull/14787